draft-ietf-secevent-delivery.xml

```diff
@@ -298,13 +291,13 @@
 
 			<t>
 			Upon receiving a SET, the Event Receiver reads the SET and validates 
-      it. The receiver MUST acknowledge receipt to the Event transmitter, using the 
+      it. The Event Receiver MUST acknowledge receipt to the Event Transmitter, using the 
```

Comment:
I would change "Receiver" to "Recipient" to match the JWT and SET specs.

Reply:
Event Transmitter and Event Receiver are the terms we settled on so far.

Do you think we should start an email thread or open an issue in the tracker to decide if we have to harmonize these terms with the SET spec?

Reply:
I'm in the midst of harmonizing them within the SET spec in response to Annabelle's comment that we are using inconsistent terms.  You'll see a proposed draft -06 with these change available for review shortly.

Annabelle pointed out some places that we are using the word "event" when what we mean is "SET".  Those will be fixed.

The "receiver" / "recipient" I believe is settled terminology.  JWT and JOSE use "recipient".  SET does (and this consistency change was reviewed by the working group before -05).  So I don't think there's a need to re-discuss it.